packages/ndla-ui/src/locale/messages-nb.ts

```diff
@@ -1184,6 +1184,8 @@ const messages = {
       firstShared: "Mappa ble delt første gang {{date}}",
       learningpathUnsupported:
         "Læringsstier og tverrfaglige caser kan ikke vises direkte i delte mapper. Dersom du trykker på lenka i navigasjonsmenyen til venstre, vil stien åpnes i en ny fane.",
```

Comment:
Denne dekker allerede to caser som ikkje kan vises direkte. Kanskje heller berre gjenbruke den og skrive: "Læringsstier, delte mapper og tverrfaglige caser ..."

Reply:
Det er et alternativ. I hovedsak er det 2 grunner te at jeg la den opp som en egen:
- På samme side som den der allerede er i bruk, brukes også denne tittelen fra ndla-frontend: `learningpathUnsupportedTitle: "Learning paths are not supported",`. Sida den er spesifikk i ordlyd til læringssti, satte jeg opp egen for mappe, og da tenkte jeg det var mer oversiktlig at det også var egen sti.
- Variabelnavn var allerede misvisende, og ville bare blitt enda mer

Et alternativ, hvis de ønskes sammenslått, mener jeg samme bør gjøres med den tilhørende i ndla-frontend, dvs noe i retning av:
- `learningpathUnsupportedTitle -> resourceUnsupportedTitle` og med ordlyd `Resource not supported` (ev ville `Resource can not be shown` vært en mer presis ordlyd)
- `learningpathUnsupported -> resourceUnsupported` og med ordlyd `Læringsstier, tverrfaglige caser og maper kan ikke ...`

Så kan det vel også tenkes at de to, som egentlig hører sammen, burde ligge på samme sted og i samme prosjekt 🤔

Reply:
Tekster i ndla-frontend er ofte lagt inn som en rask fiks på tekster som egentlig burde vore i frontend-packages. Så legg gjerne til en resourceUnsupported og resourceUnsupportedTitle og legg dei i frontend-packages, og ta denne i bruk der det trengs.